pkg/api/login_oauth.go

```diff
@@ -214,7 +214,14 @@ func (hs *HTTPServer) OAuthLogin(ctx *m.ReqContext) {
 	metrics.MApiLoginOAuth.Inc()
 
 	if redirectTo, _ := url.QueryUnescape(ctx.GetCookie("redirect_to")); len(redirectTo) > 0 {
-		ctx.SetCookie("redirect_to", "", -1, setting.AppSubUrl+"/")
+		cookie := http.Cookie{
```

Comment:
`HttpOnly: true,` missing

Reply:
I converted the line as it was before, I assume it was not needed. But I've added it.